yellowbrick/text/postag.py

```diff
@@ -167,21 +170,46 @@ def __init__(
         self.colormap = colormap
         self.colors = colors
         self.stack = stack
+        self.parser = parser
+        self.tagger = tagger
+
+    @property
+    def parser(self):
+        return self.__parser
```

Comment:
Do you think that there is a particular need for name mangling here?

Reply:
Hm, yes it's probably not necessary here. Resolved in dfd20f9 by changing to standard private variable property naming convention `_` rather than `__`.